src/main/java/org/junit/runner/Description.java

```diff
@@ -110,6 +110,18 @@ public static Description createTestDescription(String className, String name, S
         return new Description(null, formatDisplayName(name, className), uniqueId);
     }
 
+    /**
+     * Create a <code>Description</code> of a single test named <code>name</code> in the class <code>clazz</code>.
+     * Generally, this will be a leaf <code>Description</code>.
+     *
+     * @param name the name of the test (a method name for test annotated with {@link org.junit.Test})
```

Comment:
Please also document the other parameters.

Reply:
Thanks for the review, fixed!